r/tools/nixlibs.R

```diff
@@ -215,10 +215,8 @@ download_source <- function() {
 
   # Given VERSION as x.y.z.p
   p <- package_version(VERSION)[1, 4]
-  if (is.na(p) || p < 1000) {
-    # This is either just x.y.z or it has a small (R-only) patch version
-    # Download from the official Apache release, dropping the p
-    VERSION <- as.character(package_version(VERSION)[1, -4])
+  if (is.na(p)) {
```

Comment:
Do we want to remove the p < 1000 checking here? I think we want to keep that (or maybe check < 9000 since we don't want dev versions to be attempted to be downloaded, right?